ssh_key_rotator/publisher/key_publisher.py

```diff
@@ -10,6 +10,12 @@
 from ssh_key_rotator.util import get_user_path, get_username
 
 
+def _ensure_ssh_home_exists():
+    ssh_home = f"{get_user_path()}/.ssh"
+    if not os.path.isdir(ssh_home):
```

Comment:
usually the Python way is, do it first and ask for forgiveness, meaning create the directory and handle the exception

Reply:
Also for everything related to paths, shy away from `os` use `pathlib` instead please